src/watchers/html.ts

```diff
@@ -251,6 +251,44 @@ export class HTMLWatcher extends DOMWatcher<Instance> {
     };
   }
 
+  _mouseMoveEventHandler(tagname:string) {
+    return (event:MouseEvent) => {
+      let {target} = event;
+      if (!this.isInstance(target)) return;
+
+      let currentTarget:Element|null = target;
+
+      let eavs:(RawEAV|RawEAVC)[] = [];
+
+      while (currentTarget && this.isInstance(currentTarget)) {
+        let elemId = currentTarget.__element!;
+
+        if (currentTarget.listeners && currentTarget.listeners["mouse-position"]) {
```

Comment:
Let's use the same name for both the listener and the event. Either `mouse-position` or `mouse-move` seems fine.